docs/bigquery/snippets.py

```diff
@@ -458,12 +460,14 @@ def test_update_table_cmek(client, to_delete):
 
     # Set a new encryption key to use for the destination.
     # TODO: Replace this key with a key you have created in KMS.
-    updated_kms_key_name = 'projects/{}/locations/{}/keyRings/{}/cryptoKeys/{}'.format(
-        'cloud-samples-tests', 'us-central1', 'test', 'otherkey')
+    updated_kms_key_name = (
```

Comment:
FYI: I've removed the format_string variable you created. I think this is clearer to just show the string using implicit concatenation.